config/wdio/wdio.conf.js

```diff
@@ -6,12 +6,14 @@ const SeleniumDockerService = require('../../lib/wdio/services').SeleniumDocker;
 const visualRegressionConfig = require('../../lib/wdio/visualRegressionConf');
 const ServeStaticService = require('../../lib/wdio/services/index').ServeStaticService;
 const path = require('path');
+const PackageUtilities = require('lerna/lib/PackageUtilities');
+const Repository = require('lerna/lib/Repository');
```

Comment:
Should we include a `lerna` dependency for test setup?

Reply:
It should be included transitively.

Reply:
Would this have an adverse affect on non-mono-repo projects? And/or is this a dependency we want included in toolkit?

Reply:
I think the only adverse effect (since its a dev dependency) would be a longer install time